.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,9 +13,9 @@ A clear and concise description of what the bug is.
 ### General Information
 Please include the following general information about the issue and list any additional steps needed to reproduce the bug.
  - Version of the ohsome API [e.g. `1.4.0`]
- - Which API instance was requested [e.g. `https://api.ohsome.org/v1`]
+ - Which API instance was requested [e.g. remote instance, local instance with an h2 file or to the cluster connected, ohsome dashboard, etc.]
```

Comment:
IMHO it's important to know _which_ remote instance was used, so I would not skip that. I'd drop the local+cluster and dashboard examples because two examples are enough and local+cluster is a quite rare/specific set-up and the dashboard is just using to the default remote instance.

```suggestion
 - Which API instance was requested [e.g. remote instance at `https://api.ohsome.org/v1`, local instance with an h2 file, etc.]
```

Or, if you prefer to keep the third example, it would sound better if we write: “_… or connected to the cluster, …_”

Reply:
The URL should definitely be included there as an example, as it's most likely the main access point. The direct cluster connection is indeed quite specific, but still worth mentioning I guess.